server/routes/api/challenges.js

```diff
@@ -0,0 +1,43 @@
+const router = require('express').Router();
+const { Challenge } = require('../../db');
+
+/**  /api/challenges **/
+
+// get all challenges
+router.get('/', (req, res, next) => {
+  Challenge.findAll()
+    .then(challenges => res.send(challenges))
+    .catch(next);
+});
+
+// create a single challenge
+router.post('/', (req, res, next) => {
+  Challenge.create(req.body)
```

Comment:
Trusting `req.body` quite a bit here eh?

Reply:
Changed it to this:
`// create a single challenge
router.post('/', (req, res, next) => {
  const { name, description, difficulty } = req.body;
  Challenge.create({
    name,
    description,
    difficulty
  })
    .then(challenge => res.send(challenge))
    .catch(next);
});`